modules/api/test/LorisApiAuthenticatedTest.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+require_once __DIR__ .
+    "/../../../test/integrationtests/LorisIntegrationTest.class.inc";
+use GuzzleHttp\Client;
+
+/**
+ * PHPUnit class for API test suite. This script sends HTTP request to every
+ * enpoints of the api module and look at the response content, status code and
+ * headers where it applies. All endpoints are accessible at <host>/api/<version>/
+ * (e.g. the endpoint of the version 0.0.3 ofd the API "/projects" URI for the host
+ * "example.loris.ca" would be https://example.loris.ca/api/v0.0.3/projects)
+ *
+ * @category   API
+ * @package    Tests
+ * @subpackage Integration
+ * @author     Simon Pelletier <simon.pelletier@mcin.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://www.github.com/aces/Loris/
+ */
+class LorisApiAuthenticatedTest extends LorisIntegrationTest
+{
+
+    protected $client;
+    protected $headers;
+    protected $base_uri;
+
+    /**
+     * Call to LorisApiAuthenticationTest::setUp()
+     *
+     * @return void
+     */
+    public function setUp()
+    {
+        parent::setUp();
+        $this->login('UnitTester', $this->validPassword);
+    }
+
+    /**
+     * Call to LorisApiAuthenticationTest::tearDown()
+     *
+     * @return void
+     */
+    public function tearDown()
```

Comment:
I don't see any reason a method should be overridden to just call its parent, just don't override it. (This is also the case for the setup/teardown of many of the other classes in this PR.